src/static/riot/admin/server_status.tag

```diff
@@ -0,0 +1,177 @@
+<server_status>
+    <h1>Server Status</h1>
+    <div class="ui horizontal divider"></div>
+    <div class="ui relaxed three column stackable grid">
+        <div class="four wide column">
```

Comment:
Eric might have some input on aligning these? Possibly swap the gauge and the job list?